test/parallel/test-writeuint.js

```diff
@@ -33,7 +33,7 @@ const assert = require('assert');
  *  - Correctly using the offsets
  *  - Correctly interpreting values that are beyond the signed range as unsigned
  */
-function test8(clazz) {
+const test8 = (clazz) => {
```

Comment:
Hi, I think we don't have to change the global function definitions, only the function expressions passed to `assert.throws` need to be changed to arrow functions

Reply:
okay, I will overwrite this part, and I will `push -f` again.